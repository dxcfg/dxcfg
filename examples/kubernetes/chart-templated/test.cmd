npm ci
jk generate --stdout ./index.js
