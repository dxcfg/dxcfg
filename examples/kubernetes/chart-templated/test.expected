apiVersion: apps/v1
kind: Deployment
metadata:
  name: helloworld-dep
spec:
  template:
    labels:
      app: hello
      spec:
        containers:
        - image: weaveworks/helloworld:v1
          name: hello
---
apiVersion: v1
kind: Service
metadata:
  name: helloworld-svc
spec:
  selector:
    app: hello
